### ISSUE TITLE
Add `accessibilityRole` & `accessibilityStates` on View

### ISSUE THREAD

**Issue by MoOx** [reasonml-community/bs-react-native#330]
- https://facebook.github.io/react-native/docs/view#accessibilityrole
- https://facebook.github.io/react-native/docs/view#accessibilitystates

**Comment by Freddy03h** [comment]
Is there a way to deprecate accessibilityComponentType and accessibilityTraits in Reason ?

**Comment by MoOx** [comment]
Yeah I think we can do that using `[@deprecated "message"]` before something. Eg: https://github.com/reasonml-community/bs-react-native/blob/52de8a5ff8293942735e2956eee36f79ea8dc750/src/components/webView.rei#L31

**Comment by Freddy03h** [comment]
I try to add `[@deprecated "Please use accessibilityRole and accessibilityState instead"]` in `types.re` but if fail compile because of those warning

```
>>>> Start compiling
[1/65] Building src/types-BsReactNative.cmj

  Warning number 3
  (No file name)

  deprecated: accessibilityComponentType Please use accessibilityRole and accessibilityState instead

  Warning number 3
  (No file name)

  deprecated: accessibilityComponentType Please use accessibilityRole and accessibilityState instead

  Warning number 3
  (No file name)

  deprecated: accessibilityComponentType Please use accessibilityRole and accessibilityState instead

  Warning number 3
  (No file name)

  deprecated: accessibilityComponentType Please use accessibilityRole and accessibilityState instead

  Warning number 3
  (No file name)

  deprecated: accessibilityTrait Please use accessibilityRole and accessibilityState instead

  Warning number 3
  (No file name)

  deprecated: accessibilityTrait Please use accessibilityRole and accessibilityState instead

  Warning number 3
  (No file name)

  deprecated: accessibilityTrait Please use accessibilityRole and accessibilityState instead

  Warning number 3
  (No file name)

  deprecated: accessibilityTrait Please use accessibilityRole and accessibilityState instead

  We've found a bug for you!
  /Users/freddy/workspace/bs-react-native/src/types.re

  Some fatal warnings were triggered (8 occurrences)

>>>> Finish compiling(exit: 1)
```

**Comment by MoOx** [comment]
I think that warnings are considered as error on CI. Not sure what I can do.
Maybe we could try to not use those in our codebase? I don't think we can't really since we are using this named arguments passed down to view props...
You know what, to make this simple, just comment deprecated for the moment & we will move forward with this later (new issue?)
